### PR TITLE
Updated object returned by groups/:id/members calls

### PR DESCRIPTION
**Pull request by pschwartz**
Gitlab MR for Object addition for created_at: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/45156

Signed-off-by: Philip Marc Schwartz <Philip.Schwartz1@T-Mobile.com>

### CODE REVIEW

**Comment by pschwartz** [comment]
@svanharmelen Any chance at a review?

File: group_members_test.go [hunk 1]
```diff
@@ -48,3 +49,49 @@ func TestListBillableGroupMembers(t *testing.T) {
...
+
+func TestListGroupMembersWithoutSAML(t *testing.T) {
+	mux, server, client := setup(t)
+	defer teardown(server)
+
+	mux.HandleFunc("/api/v4/groups/1/members",
+		func(w http.ResponseWriter, r *http.Request) {
+			testMethod(t, r, http.MethodGet)
+			fmt.Fprint(w, `[{"id": 1,"username": "raymond_smith","name": "Raymond Smith","state": "active","avatar_url": "https://www.gravatar.com/avatar/c2525a7f58ae3776070e44c106c48e15?s=80&d=identicon","web_url": "http://192.168.1.8:3000/root","created_at": "2012-10-21T14:13:35Z","expires_at": "2012-10-22T14:13:35Z","access_level": 30,"group_saml_identity": null}]`)
+		})
```
Comment (svanharmelen) [inline group_members_test.go:61]: Can you format these so it's a bit more readable?

File: group_members.go [hunk 2]
```diff
@@ -43,14 +44,16 @@ type GroupMemberSAMLIdentity struct {
 // GroupMember represents a GitLab group member.
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/members.html
+// Gitlab MR for Object addition for created_at: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/45156
```
Comment (svanharmelen) [inline group_members.go:47]: Please remove this line...

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Looks good, but have two small comments/requests...

File: group_members_test.go (hunk 1 above)
Comment (pschwartz) [reply group_members_test.go:96]: Updated with commit.

File: group_members.go (hunk 2 above)
Comment (pschwartz) [reply group_members.go]: Removed with commit.

**Review by svanharmelen — APPROVED** [review]
Super nice, thanks 👍🏻

**Pull request merged**